.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,68 @@
 <!--
 Thanks for sending a pull request!
 
-Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/master/CONTRIBUTING.md#submitting-pull-requests).
+Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).
 
 In case you're only changing docs, make sure to prefix the pull-request title with "[CI:DOCS]".  That will prevent functional tests from running and save time and energy.
+
+Finally, be sure to sign your commits (e.g. `git commit -s`)
+-->
+
+
+#### What type of PR is this?
+
+<!--
+Please label this pull request according to what type of issue you are
+addressing, especially if this is a release targeted pull request.
+
+Uncomment only one `/kind <>` line, hit enter to put that in a new line, and
+remove leading whitespace from that line:
 -->
+
+> /kind api-change
+> /kind bug
+> /kind cleanup
+> /kind deprecation
+> /kind design
+> /kind documentation
+> /kind failing-test
+> /kind feature
+> /kind flake
+> /kind other
+
+#### What this PR does / why we need it:
+
+#### How to verify it
+
+#### Which issue(s) this PR fixes:
+
+<!--
+Automatically closes linked issue when PR is merged.
+Uncomment the following comment block and include the issue
+number or None on one line.
+Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`, or `None`.
+-->
+
+<!--
+Fixes #
+or
+None
+-->
+
+#### Special notes for your reviewer:
+
+#### Does this PR introduce a user-facing change?
+
+<!--
+If no, just write `None` in the release-note block below. If yes, a release note
+is required: Enter your extended release note in the block below. If the PR
+requires additional action from users switching to the new release, include the
+string "action required".
+
+For more information on release notes please follow the kubernetes model:
+https://git.k8s.io/community/contributors/guide/release-notes.md
+-->
+
+```release-note
+
+```
```

Comment:
Is the lack of a release note entry something we could fail in the smoke test?

Reply:
Yes in fact we can!  The entire PR Description is available as an env. var for us to do what we please with.  The challenge is in coming up with the right script to do it :grin:

Reply:
Clarification: I'm willing to take on this work...after this PR is finalized and merged or closed.